src/apify_shared/models.py

```diff
@@ -1,4 +1,6 @@
-from typing import Dict, Generic, List, TypeVar
+from __future__ import annotations
```

Comment:
I like this and I'm not against it. Not sure if something could go wrong with this. But if lint and tests pass, I guess it's fine.

Reply:
Yeah... linting and tests have passed, I personally tested it on Python 3.8, and it's also utilized in larger projects like Airflow (e.g. here [airflow/providers/sftp/operators/sftp.py#L19](https://github.com/apache/airflow/blob/0a93e2e28baa282e20e2a68dcb718e3708048a47/airflow/providers/sftp/operators/sftp.py#L19)), which supports Python 3.8 and follows a similar approach.

So hopefully we should be good :crossed_fingers:.